src/helpers/reporthelper.js

```diff
@@ -80,6 +80,7 @@ const reporthelper = function () {
           weeklySummaryOption: 1,
           adminLinks: 1,
           bioPosted: 1,
+          role: 1,
```

Comment:
I always feel that the app is very laggy these days. Adding `role: 1` in very weekly report will also slow down the app. I suggest query this field separately when required.